test/integration/peggy-e2e-test.md

```diff
@@ -12,46 +12,63 @@ sifnoded start
 
 1. open new console
 cd smart-contracts
-ebrelayer init tcp://localhost:26657 ws://localhost:7545/ 0x30753E4A8aad7F8597332E813735Def5dD395028 user1 --chain-id=sifchain
+
+ebrelayer init tcp://localhost:26657 ws://localhost:7545/ \
+0xdDA6327139485221633A1FcD65f4aC932E60A2e1 \ # BridgeRegistry contract address
+akasha <akasha's mnemonic> --chain-id=sifchain
 
 ### case 1: lock eth and send to cosmos user2 from eth operator account
 1. check the balance of operator before lock
-yarn peggy:getTokenBalance  0x627306090abaB3A6e1400e9345bC60c78a8BEf57 eth
+yarn peggy:getTokenBalance \
+0xf17f52151EbEF6C7334FAD080c5704D77216b732  \ # any ethereum account
+eth
 2. check the ballance of contract before lock
-yarn peggy:getTokenBalance  0x2C2B9C9a4a25e24B174f26114e8926a9f2128FE4  eth
+yarn peggy:getTokenBalance \
+0x75c35C980C0d37ef46DF04d31A140b65503c0eEd \ # BridgeBank contract address
+eth
 3. check the user balance before lock
-sifnodecli query account $(sifnodecli keys show user2 -a)
+sifnodecli query account $(sifnodecli keys show akasha -a)
 
-yarn peggy:lock $(sifnodecli keys show user2 -a) 0x0000000000000000000000000000000000000000 1000000000000000000
+yarn peggy:lock $(sifnodecli keys show akasha -a) 0x0000000000000000000000000000000000000000 100 0xf17f52151EbEF6C7334FAD080c5704D77216b732
 
-4. check the balance of operator before lock
-yarn peggy:getTokenBalance  0x627306090abaB3A6e1400e9345bC60c78a8BEf57 eth
-5. check the ballance of contract before lock
-yarn peggy:getTokenBalance  0x2C2B9C9a4a25e24B174f26114e8926a9f2128FE4  eth
-6. check the user2 balance before lock
-sifnodecli query account $(sifnodecli keys show user2 -a)
+4. check the balance of operator after lock
+yarn peggy:getTokenBalance 0xf17f52151EbEF6C7334FAD080c5704D77216b732 \ # any ethereum account
+eth
+5. check the ballance of contract after lock
+yarn peggy:getTokenBalance 0x75c35C980C0d37ef46DF04d31A140b65503c0eEd \ # BridgeBank contract address
+eth
+6. check the cosmos user balance after lock
+sifnodecli query account $(sifnodecli keys show akasha -a)
 
-### case 2: burn user2's eth in cosmos then asset to back to ethereum's validator account
-1. check the validator's balance before burn
-yarn peggy:getTokenBalance 0xf17f52151EbEF6C7334FAD080c5704D77216b732 eth
-sifnodecli query account $(sifnodecli keys show user2 -a)
+### case 2: burn user's ceth in cosmos then asset to back to ethereum's receiver account
```

Comment:
"then asset to back to ethereum's receiver account" - I'm not sure what this means

Reply:
not sure either. I have fixed it.